CONTRIBUTING.md

```diff
@@ -515,6 +516,62 @@ this API we will not break anyone, because they weren't able to use it. The file
 `allowed-breaking-changes.txt` in the root of the repo is an exclusion file that
 can be used in these cases.
 
+### Examples
+
+Examples typed in fenced code blocks (looking like `'''ts`, but then with backticks
+instead of regular quotes) will be automatically extrated, compiled and translated
+to other languages when the bindings are generated.
+
+To successfully do that, they must be compilable. The easiest way to do that is using
+a *fixture*, which looks like this:
+
+```
+'''ts fixture=with-bucket
+bucket.addLifecycleTransition({ ... });
+'''
+```
+
+While processing the examples, the tool will look for a file called
+`rosetta/with-bucket.ts-fixture` in the package directory. This file will be
```

Comment:
Perhaps a bit late now but would be better to have used the extension format `.ts.fixture` in line with how template files are usually named like `.html.erb`, `.template.json`, etc.

Reply:
That should be `.fixture.ts` then. I recall not doing that because TSC will want to compile it then, something like that.

Reply:
Hence the suggestion to go `.ts.fixture`. 

Rationale: The file might not be strictly typescript (or atleast not a complete typescript app). When the fixture is resolved (with the example code), it becomes correct typescript (or fully valid).

Non-blocking.